Change missing or empty secret error message (#2074)
cyberark/conjur: CHANGELOG.md

### Changed
- The secrets batch retrieval endpoint now refers to the `Accept-Encoding` header rather than `Accept` to determine the response encoding
  [cyberark/conjur#2065](https://github.com/cyberark/conjur/pull/2065)

> - Return a proper error message when trying to fetch a missing or empty secret

Reviewer: ```suggestion
- Return a proper error message when trying to fetch a missing or empty secret.
```

Reviewer 2: ```suggestion
- When trying to fetch a missing or empty secret, a proper error message is returned
```

### Changed
- The secrets batch retrieval endpoint now refers to the `Accept-Encoding` header rather than `Accept` to determine the response encoding
  [cyberark/conjur#2065](https://github.com/cyberark/conjur/pull/2065)

> - When trying to fetch a missing or empty secret, a proper error message is now returned

Reviewer: ```suggestion
- When trying to fetch a missing or empty secret, a proper error message is now returned.
```
